assessment/asr_validation/wald2017B/analysis/plots/ASR_validation_plotting.py

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/env python3
+# /*************************************************/
+# /*           DO NOT MODIFY THIS HEADER           */
+# /*                                               */
+# /*                 BlackBear                     */
+# /*                                               */
+# /*    (c) 2020 Battelle Energy Alliance, LLC     */
+# /*            ALL RIGHTS RESERVED                */
+# /*                                               */
+# /*   Prepared by Battelle Energy Alliance, LLC   */
+# /*     Under Contract No. DE-AC07-05ID14517      */
+# /*     With the U. S. Department of Energy       */
+# /*                                               */
+# /*     See COPYRIGHT for full restrictions       */
+# /*************************************************/
+"""Create all plots for asr_validation/wald2017b assessment case."""
+import pandas as pd
+import matplotlib.pyplot as plt
+
+# Plot Settings
+plt.style.use('../../../../../scripts/plot_style.mplstyle')
+
+# Global Constants
+AXES = ["x", "y", "z"]
+SEQ = range(1, 7)
```

Comment:
This should have the option to select different no. of points in each direction.